src/lib/provable/gates.ts

```diff
@@ -284,6 +286,13 @@ function raw(kind: KimchiGateType, values: Field[], coefficients: bigint[]) {
   );
 }
 
+function addRuntimeTableConfig(id: number, first_column: bigint[]) {
+  Snarky.gates.addRuntimeTableConfig(
```

Comment:
if i remember correctly, the `id` of the runtime tables must be anything but 1 since that's reserved. I think we should make the developer aware (I am not sure what would happen right now if we specify a runtime table with id 1) - I think in the doc comment is fine for now

Reply:
Yeah actually 0 and 1 iirc... What about asserting that id > 1? I haven't seen any of those functions performing checks...